README.md

```diff
@@ -17,6 +17,29 @@ generate: wag-generate-deps
 	$(call wag-generate,./swagger.yml, $(PKG))
 ```
 
+Define global BadRequest and InternalError response types. These are used internally for validation errors and unknown errors respectively. They must reference a definition with a msg field. For example:
+
+```
+responses:
+  BadRequest:
+    description: Bad Request
+    schema:
+      $ref: "#/definitions/Error"
+  InternalError:
+    description: Internal Error
+    schema:
+      $ref: "#/definitions/Error"
+
+definitions:
+  Error:
+    type: object
+    properties:
+      msg:
+        type: string
+```
+
+If your yml doesn't define a 400 or a 500 response for any of your operations, Wag will implicitly set it to the globally defined response.
```

Comment:
how does wag identify these? If it's by name, e.g. `BadRequest`, `InternalError`, then I'd mention that here.